src/rewriter.coffee

```diff
@@ -512,18 +512,19 @@ exports.Rewriter = class Rewriter
       return 1 if     token[2]
       return 1 unless token.generated or token.explicit
       if token[0] is '{' and nextLocation=tokens[i + 1]?[2]
-        {first_line: line, first_column: column, range} = nextLocation
+        {first_line: line, first_column: column, range: [rangeIndex]} = nextLocation
       else if prevLocation = tokens[i - 1]?[2]
-        {last_line: line, last_column: column, range} = prevLocation
+        {last_line: line, last_column: column, range: [, rangeIndex]} = prevLocation
+        column += 1
```

Comment:
This is another place where we could suffer from "column + 1" being not always accurate (ie where we really want to know whether this token ends with a newline and thus incrementing its `last_line`/`last_column` should give us the first column of the next line, not the next column in the current line)

I tried playing around with tracking this when creating tokens so that we could correctly handle cases where "column + 1" isn't correct. It seemed to be working, but I didn't see any current use cases to test against where a token/node that gets used for AST location data ends with an included newline. So I saved that branch and can revisit it if we see actual test cases where column + 1 is causing inaccuracy in AST location data